build/geloofsverdediging/Aquino_Summa_01.txt

```diff
@@ -310,24 +310,24 @@ Daar is echter tegen aan te voeren, dat elke bijkomstigheid in een subject is. W
 1e) In verband met wat hierboven werd uiteengezet. In God is immers geen samenstelling van kwantitatieve delen, daar Hij geen lichaam is; ook is er geen samenstelling van stof en vorm; daarbij is zijn natuur één met zijn persoon, en zijn wezenheid met zijn bestaan; samenstelling van geslacht en verschil, van subject en bijkomstigheden is er ook niet in Hem. Hij is dus op geen enkele wijze samengesteld, maar geheel enkelvoudig.
-2e) Al wat samengesteld is komt na de samenstellende delen en hangt er van af. Welnu God is het eerste wezen, zoals hierboven werd bewezen (IIe Kw., 3e Art.).
-3e) Al wat samengesteld is heeft een oorzaak, omdat dingen die uit zich zelf verschillen, alleen kunnen samenkomen door de werking van een oorzaak die ze verenigt. God echter, we bewezen het reeds (IIe Kw., 3e Art.), heeft geen oorzaak, daar Hij zelf de eerste werkende oorzaak is.
-4e) In al wat samengesteld is, is er aanleg en werkdadelijkheid: ofwel is een der deel en akt tegenover een ander, ofwel zijn minstens al de delen in potentie tegenover het geheel. Dit nu heeft niet plaats bij God.
-5e) Ieder samengesteld wezen heeft een hoedanigheid die met toekomt aan een der delen. Dit blijkt vooreerst in een geheel dat samengesteld is uit ongelijke delen. Geen enkel deel van de mens toch is een mens, en geen enkel deel van de voet is een voet. Wat nu een geheel betreft dat uit gelijke delen is samengesteld, zijn er wel dingen die én van het geheel, én van de delen gezegd worden: een deel der lucht is lucht, een deel van het water is water. Toch wordt er iets van het geheel gezegd wat niet op de delen toepasselijk is: indien al het water bijeen twee maten meet, dan is dit niet toepasselijk op een deel er van. Zo is er inderdaad in elk samengesteld wezen iets, wat er vreemd aan is. Maar hoewel nu eveneens kan gezegd worden van een ding dat een vorm heeft, dat het iets vreemds heeft (in een wit voorwerp kan er b. v. iets anders zijn dan wit), toch geldt dit niet van de vorm zelf. Daarom kan God op generlei wijze samengesteld zijn, omdat Hij zuivere vorm is, of beter zuiver zijn. En dit bewijs vinden we ook bij Hilarius, VIIe Boek Over de Drie-eenheid (Nr 27), waar hij zegt: “God die een kracht is, is niet gemaakt uit zwakheid; Hij die licht is, is niet samengesteld uit duisternis”.
+2e) Al wat samengesteld is komt na de samenstellende delen en hangt ervan af. Welnu God is het eerste wezen, zoals hierboven werd bewezen (IIe Kw., 3e Art.).
+3e) Al wat samengesteld is heeft een oorzaak, omdat dingen die uit zichzelf verschillen, alleen kunnen samenkomen door de werking van een oorzaak die ze verenigt. God echter, we bewezen het reeds (IIe Kw., 3e Art.), heeft geen oorzaak, daar Hij zelf de eerste werkende oorzaak is.
+4e) In al wat samengesteld is, is er aanleg en werkdadigheid: ofwel is een der delen en akt tegenover een ander, ofwel zijn minstens al de delen in potentie tegenover het geheel. Dit nu heeft niet plaats bij God.
```

Comment:
Blijkbaar een typfout in het originele PDF bestand? (delen vs deel)

<img width="1245" height="178" alt="Screenshot 2025-08-31 at 10 29 55" src="https://github.com/user-attachments/assets/a81d777e-d0ad-44d5-8977-6d8afafd1641" />